addons/postcss/src/index.ts

```diff
@@ -0,0 +1,31 @@
+import { Configuration } from 'webpack';
+import { logger } from '@storybook/node-logger';
+
+// TODO(blaine): Should we take PostCSS config in plugin options?
+export const webpack = (webpackConfig: Configuration = {}): Configuration => {
+  // TODO(blaine): Should we take a perf hit to log the PostCSS version?
```

Comment:
yes pls!

Reply:
or maybe not if you need to load something in order to do it 🤔

Reply:
PostCSS will always need to be loaded, and v7 and v8 include a version property on the Processor instance. If we rely on peerDep exclusively, then we wouldn't want to require postcss ourselves. I'll follow up with a further exploration as a PR comment.